src/main/java/io/jenkins/plugins/monitoring/MonitorPortletFactory.java

```diff
@@ -1,14 +1,28 @@
 package io.jenkins.plugins.monitoring;
 
+import hudson.Extension;
 import hudson.ExtensionPoint;
 import hudson.model.Run;
 
 import java.util.Collection;
+import java.util.Collections;
 
 /**
- * Factory for {@link MonitorPortlet}.
+ * <p>Defines the {@link ExtensionPoint} to register one or more new pull request monitoring portlet.</p>
+ *
+ * <p>The {@link #getDisplayName()} is shown in a dropdown list as optgroup. The children of the optgroup
+ * are the registered portlets of {@link #getPortlets(Run)}. </p>
+ *
+ * <p>Since an empty dashboard is always added by default, it is possible that the method {@link #getPortlets(Run)} will
```

Comment:
Another topic to discuss: should we show some portlets by default?

Reply:
For documentation purposes only: Add new method `isDefault()` to abstract class. With version 1.6.0 it will be released but the flag is still ignored by the plugin. The functionality has to be implemented.